src/visualizer/raysAppFunctions/updateScenario.m

```diff
@@ -164,6 +236,50 @@ function extractQdFilesInfo(app)
                 .qdInfo(out.TX,out.RX) = out;
         end
     end
+    else
+        qdOutput = readQdTargetJsonFile(sprintf('%s/QdFiles/qdOutput.json',...
```

Comment:
correct. It is in src/utils. If you run visualizer, your path should be src folder. e.g., C:\Users\nnv2\Documents\GitHub\qd-realization-case\src

Reply:
I also set path as C:\Users\nnv2\Documents\GitHub and it is working at my end. Could you please check again and send me the exact error

Reply:
my bad. it was using other folder for these missing functions. I added missing functions now. Sorry

Reply:
Thanks! Now it is working.
I have only one question. The `readQdTargetJsonFile` looks very similar to `readQdJsonFile`: if I understand correctly, the only difference is in the capital letters in the json files and thus in the script. Is there any chance we can use the same notation for both jsons, and thus use only `readQdJsonFile`? In this case, we could avoid duplicating a piece of code that does the same job, thus improving maintainability and clarity, and have a more consistent notation. What do you think about this?

Reply:
No, we need this because I don't want to trouble Tanguy for his visualizer. At this moment, target scenario and non-target scenario have two different notations when writing the output QD file which is somehow came from Steve PR. Maybe he did this intentionally because of Tanguy's visualizer.

Reply:
Alright, thanks for the explanation!